public_html/wp-content/plugins/pattern-translations/pattern-translations.php

```diff
@@ -32,21 +32,24 @@
  * Creates or updates a localised pattern.
  */
 function create_or_update_translated_pattern( Pattern $pattern ) {
+	$parent = get_post( $pattern->parent->ID );
+
 	$args = [
 		'ID'           => $pattern->ID,
 		'post_type'    => POST_TYPE,
 		'post_title'   => $pattern->title,
 		'post_name'    => $pattern->ID ? $pattern->name : ( $pattern->name . '-' . $pattern->locale ), // TODO: Translate the slug?
-		'post_date'    => $pattern->parent ? get_post( $pattern->parent->ID )->post_date : '',
+		'post_date'    => $pattern->parent ? $parent->post_date : '',
 		'post_content' => $pattern->html,
 		'post_parent'  => $pattern->parent ? $pattern->parent->ID : 0,
-		'post_author'  => $pattern->parent ? get_post( $pattern->parent->ID )->post_author : 0,
-		'post_status'  => $pattern->parent ? get_post( $pattern->parent->ID )->post_status : 'pending',
-		'meta_input'   => [
+		'post_author'  => $pattern->parent ? $parent->post_author : 0,
```

Comment:
I'm tempted to suggest we could just change these to `$parent->post_author ?? 0` rather than the ` a ? b : c` operator.

Reply:
Good idea. Changed in 880ea43